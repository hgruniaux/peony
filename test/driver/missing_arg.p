//# RUN: peony -fmax-errors -fdiagnostics-column-origin= -o
//~! ERROR: missing argument to '-fmax-errors'
//~! ERROR: missing argument to '-fdiagnostics-column-origin'
//~! ERROR: missing argument to '-o'
