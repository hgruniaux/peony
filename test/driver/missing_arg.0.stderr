error: missing argument to '-fmax-errors'
error: missing argument to '-fdiagnostics-column-origin'
error: missing argument to '-o'
